4_arm.ipynb

```diff
@@ -44,13 +84,26 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "アームとして定義されている各関節の名称を配列として取得します。以降の各コマンドでは、この配列の順序に対応させて各関節値を設定します。\n",
     "\n",
     "ここで`arm`はロボットのアームの関節角を管理する変数です。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Get the name of the joints defined as arm group as an array. The subsequent commands set joint values according to the order of this array.\n",
```

Comment:
"Get the name of the joints defined as the arm group as an array. The subsequent commands set joint values according to the order of this array.\n",